### DISCUSSION TITLE
How do you adjust the rotation and size at the same time?

### DISCUSSION THREAD

**Discussion by hjaeho95** [sceneview/sceneview#92] [Q&A] [answered]
Hello, we are testing the ar-cursor-placement on the sample.
Although "editableTransforms = setOf(EditableTransform.SCALE, EditableTransform.ROTATION)" is used, only the scale can be adjusted.
You can also adjust rotates by changing to "editableTransforms = setOf (EditableTransform.ROTATION)".
I want to adjust the size and rotation at the same time.
How shall I do it?

**Comment by grassydragon** [comment, marked as answer]
Hi!
It looks like a problem with the latest version of SceneView.

**Reply by hjaeho95** [reply]
Thank you, I have confirmed that it works in version 0.5.1!!
